xilem/examples/calc.rs

```diff
@@ -187,6 +187,12 @@ impl Calculator {
 const DISPLAY_FONT_SIZE: f32 = 30.;
 const GRID_GAP: f64 = 2.;
 fn app_logic(data: &mut Calculator) -> impl WidgetView<Calculator> {
+    let num_row = |nums: [&'static str; 3], operator| {
+        flex_row((
+            nums.map(|num| digit_button(num).flex(1.)),
```

Comment:
This is slightly more complex to avoid 2 additional lines, I would in practice rather use a little bit of repetition here, but I think it shows another interesting aspect, when it's not just an array with 3 elements...

Reply:
I think this change should use a different free component, since this doesn't actually use any state from within the function justifying being a closure.

Reply:
I thought about it, but than it adds more boilerplate to add all the typing, do you think that's worth it anyway?